### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by Ambercroft** [Bodmer/TFT_eSPI#870] [General]
I just started playing with the Wiki on my site. https://github.com/Ambercroft/TFT_eSPI 
I copied this project there so I could build the documentation and learn more how to use the features here. Once it is in a suitable state it can be merged.
I started with the keywords.txt to build the library of functions.

Suggestions / edits welcome. 

Bob

**Comment by Bodmer** [comment]
Looks promising!

**Comment by Bodmer** [comment]
There is some [function information here](https://github.com/Bodmer/User_Manual_TFT_eSPI/blob/master/TFT_eSPI.cpp.All.pdf) that was generated by a user but unfortunately is not in an editable form.

**Reply by Ambercroft** [reply]
I will merge the info into the work I am doing.

**Comment by Ambercroft** [comment]
The documentation is looking better. Lots to do.  [https://github.com/Ambercroft/TFT_eSPI](https://github.com/Ambercroft/TFT_eSPI) 
@Bodmer added what I could from the PDF.

**Comment by Ambercroft** [comment]
Back to work now so editing the docs is slower. Working through each section created. Table on the first page shows what I'm working on and what I see for next steps.

**Comment by Bodmer** [comment]
Hi Bob, thank you for your continued support. The Wiki is developing nicely.

Most questions I get are related to hardware and connections, so I will write something on that aspect.
